packages/gasket-intl/src/with-intl-provider.js

```diff
@@ -0,0 +1,114 @@
+import React, { useReducer } from 'react';
+import hoistNonReactStatics from 'hoist-non-react-statics';
+import merge from 'lodash.merge';
+import { IntlProvider } from 'react-intl';
+import { GasketIntlContext } from './hooks';
```

Comment:
Your context is part of the `hooks` file? Should it be somewhere else that both places can pull from?